docs/get_started.rst

```diff
@@ -1,17 +1,353 @@
+| Which [source format] do you want to use?                        | Metadata API format is the "older" format and stores data under the `src/` directory.                                                                                                                                                                                                                 |
+|                                                                  | DX source format (aka "SFDX Format") stores data under the `force-app/` directory.                                                                                                                                                                                                                    |
++------------------------------------------------------------------+-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+| Are you extending another CumulusCI project such as NPSP or EDA? | CumulusCI makes it easy to build extensions of other projects configured for CumulusCI like Salesforce.org's NPSP and EDA.  If you are building an extension of another project using CumulusCI and have access to its Github repository, use this section to configure this project as an extension. |
++------------------------------------------------------------------+-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+| Default Branch                                                   | In GitHub, what is your projects master/main brach? Defaults the branch that is currently checked out in your local repository.                                                                                                                                                                       |
++------------------------------------------------------------------+-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+| Feature Branch Prefix                                            | In GitHub, do you use a branch prefix you use for feature branches? Defaults to 'feature/'.                                                                                                                                                                                                           |
```

Comment:
All these descriptions for the branch prefixes are not grammatical

Reply:
Jeez, hopefully this was after a long day of doc writing. lol.